src/overlay/Tracker.cpp

```diff
@@ -28,6 +28,9 @@ Tracker::Tracker(Application& app, Hash const& hash, AskPeer& askPeer)
     , mItemHash(hash)
     , mTryNextPeer(app.getMetrics().NewMeter(
           {"overlay", "item-fetcher", "next-peer"}, "item-fetcher"))
+    , mFetchItemDuration(
```

Comment:
let's make this
* type aware, ie `overlay.fetch.txset` and `overlay.fetch.qset`
* track successful downloads only (instead of including cancels) - a new method needs to be added for this

Reply:
updated